### DISCUSSION TITLE
SMTP configuration not working

### DISCUSSION THREAD

**Discussion by Rishabhh17** [chatwoot/chatwoot#5946] [Self hosted] [answered]
Hello @vishnu-narayanan @sojan-official @pranavrajs I have tried every possible way to add my smtp credentials but not able tpo add agent in chatwoot. I am struggling since so long.  Below are the logs of sidekiq.

[Sidekiq worker](https://justpaste.it/9qlp4)

Office365 - SMTP

**Comment by sojan-official** [comment, marked as answer]
@Rishabhh17  could you try the steps mentioned here ? 
https://www.chatwoot.com/docs/self-hosted/faq#how-to-debug-smtp-email-errors-in-chatwoot

we don't have official support for office365, we recommend a provider like sendgrid, mailgun , ses etc for the SMTP configuration as described in the docs
https://www.chatwoot.com/docs/self-hosted/configuration/environment-variables#configure-emails
